frontend-svelte/src/components/widgets/inputs/MenuBarInput.svelte

```diff
@@ -95,7 +92,7 @@
 				{/if}
 			</div>
 			{#if entry.children && entry.children.length > 0}
-				<MenuList open={entry.ref?.menuIsOpen() || false} entries={entry.children || []} direction="Bottom" minWidth={240} drawIcon={true} bind:this={entry.ref} />
+				<MenuList on:open={(e) => entry.ref.isOpen=e.detail} open={entry.ref?.isOpen || false} entries={entry.children || []} direction="Bottom" minWidth={240} drawIcon={true} bind:this={entry.ref} />
```

Comment:
Did `menuIsOpen()` not work? And how is it possible to directly get the `.isOpen` value from the component? I thought the component has to be configured to compile with a getter/setter for that to work.

Could you please fix the formatting by surrounding the `=` with spaces? Thanks.

Reply:
It just works like this. The component object is behind a Proxy, so get/set will trigger update on that `export let`

Reply:
I did some further digging in the Vue version and, as far as I can tell, this does seem to be dead code